src/logic/__tests__/validateCurrentProject.test.js

```diff
@@ -0,0 +1,56 @@
+import validateCurrentProject from '../validateCurrentProject';
+
+import Analyzer from '../../analyzers';
+import validateSource from '../helpers/validateSource';
+import {getCurrentProject} from '../../selectors';
+
+const mockHtmlSource = '<html></html>';
+const mockCssSource = 'div {}';
+
+jest.mock('../../analyzers');
+jest.mock('../helpers/validateSource');
```

Comment:
So this I think may be going a little further than we want with mocking—in particular the fact that `validateCurrentProject` delegates some of its logic to the `validateSource` helper is essentially an implementation detail of `validateCurrentProject`; if in the future we decided to pull all of that logic directly into `validateCurrentProject`, I don’t think we would consider that breaking from the standpoint of the unit tests.

So my proposal would be to just test the behavior of the `validateCurrentProject` logic here, including any behavior that happens to be implemented in `validateSource`; and remove the unit test for the `validateSource` helper.